src/anim/evaluator/anim-clip.js

```diff
@@ -7,11 +7,11 @@ import { AnimSnapshot } from './anim-snapshot.js';
  * @classdesc AnimClip wraps the running state of an animation track. It contains and update
  * the animation 'cursor' and performs looping logic.
  * @description Create a new animation clip.
- * @param {AnimTrack} track - the animation data.
- * @param {number} time - the initial time of the clip.
- * @param {number} speed - speed of the animation playback.
- * @param {boolean} playing - true if the clip is playing and false otherwise.
- * @param {boolean} loop - whether the clip should loop.
+ * @param {AnimTrack} track - The animation data.
+ * @param {number} time - The initial time of the clip.
+ * @param {number} speed - Speed of the animation playback.
+ * @param {boolean} playing - True if the clip is playing and false otherwise.
```

Comment:
If it's a value, should it stay lower case and be wrapped in `` `? (if that works?)

Reply:
Done.